src/System.IO.FileSystem/tests/Directory/Delete.cs

```diff
@@ -204,6 +205,17 @@ public void UnixShouldBeAbleToDeleteHiddenDirectory()
             Assert.False(Directory.Exists(testDir));
         }
 
+        [Fact]
+        [OuterLoop("Needs sudo access")]
+        [PlatformSpecific(TestPlatforms.Linux)]
```

Comment:
Linux only problem, Mac does not have this functionality in mount so there's no similar test to be created as far as I know.